frontend/src/modules/entitydetails/components/EntityDetails.test.js

```diff
@@ -194,8 +196,17 @@ describe('<EntityDetailsBase>', () => {
     });
 });
 
-describe('<EntityDetails>', () => {
+/**
+ * This test ends up calling fetch(), and expects it to actually work.
+ * It needs to be skipped for now, because it triggers async errors when
+ * the API calls fail, and the calling code doesn't handle the errors.
```

Comment:
At the moment, skipped tests are showing up in the test suite results. I'll file issues for these after this burst of PRs has landed, so that it's easier to point at the skipped test in the code.